db/schema.rb

```diff
@@ -88,6 +88,19 @@
   add_index "bundle_products", ["bundle_product_id"], name: "fk_bundle_prod_prod", using: :btree
   add_index "bundle_products", ["product_id"], name: "fk_bundle_prod_bundle", using: :btree
 
+  create_table "card_readers", force: :cascade do |t|
```

Comment:
What do you think of collapsing the two tables into one? If you really want to do that as a separate PR, that's fine. It just seems like that would be extra work as you would then need to write data migrations as opposed to just schema changes.